### ISSUE TITLE
Mac JavaFX not working or available on startup

### ISSUE THREAD

**Issue by feordis7** [DraqueT/PolyGlot#651]
I am using MacOS High Sierra v10.13.6
When I attempt to start PolyGlot.jar (Polyglot 2.3.3), the Polyglot logo/startup screen appears, then it bring up the following warning:

"Unable to load Java FX. Download and install to use Polyglot (JavaFX not included in some builds of Java 8 for Linux)."

**Comment by DraqueT** [comment]
Glad that worked for you! I was about to suggest that you try the installer but you found it on your own. Please let me know if you run into any other trouble! :3

**Comment by feordis7** [comment]
No, it did not work. The error happened after I installed the Java. I don't
know whether or not it came with JavaFX, or what that is.

**Comment by DraqueT** [comment]
Well shoot. Gonna look into this one a bit. Thanks for letting me know it hadn't been resolved.

**Comment by DraqueT** [comment]
Ok. I see what the issue is! You've installed the JDK, which is the Java Development Kit. You need to install the JVM, which can be found here: https://www.java.com/en/download/mac_download.jsp

Please let me know if this helps!

**Comment by feordis7** [comment]
It works now! Thank you!
